lib/profile/header.dart

```diff
@@ -1,63 +1,141 @@
+            buildProfilePhoto(),
+            buildFullName(),
+            buildPopUpMenuButton(context)
+          ]
+        )
+      ),
+    );
+  }
+
+  buildProfilePhoto() {
+    return Container(
+      width: 60.0, 
+      height: 60.0, 
+      decoration: BoxDecoration(
+        color: kSecondaryColor, 
+        shape: BoxShape.circle
+      )
+    );
+  }
+
+  buildFullName() {
+    return Container(
+      margin: const EdgeInsets.only(top: 10.0),
       child: Column(
         mainAxisAlignment: MainAxisAlignment.center,
-        children: <Widget> [
-          Container(
-            width: 60.0, 
-            height: 60.0, 
-            decoration: BoxDecoration(
-              color: Color(0xFF45CAB9), 
-              shape: BoxShape.circle
-            )
-          ),
-          Container(
-            margin: const EdgeInsets.only(top: 10.0),
-            child: Column(
-              mainAxisAlignment: MainAxisAlignment.center,
-              children: <Text> [
-                Text(
-                  fullName,
-                  style: GoogleFonts.muli(
-                    textStyle: TextStyle(
-                      color: Colors.white, 
-                      letterSpacing: .5, 
-                      fontSize: 30.0, 
-                      fontWeight: FontWeight.bold
-                    ), 
-                  )
-                ), 
-                Text(
-                  emailAddress, 
-                  style: GoogleFonts.muli(
-                    textStyle: TextStyle(
-                      color: Colors.white,  
-                      letterSpacing: .5, 
-                      height: 1.5, 
-                      fontSize: 14.0, 
-                    ), 
-                  )
-                )
-              ]
+        children: <Text> [
+          Text(
+            fullName,
+            style: GoogleFonts.muli(
+              textStyle: TextStyle(
+                color: Colors.white, 
+                letterSpacing: .5, 
+                fontSize: 20.0, 
+                fontWeight: FontWeight.bold
+              ), 
             )
           )
         ]
+      )
+    );
+  }
+
+  buildPopUpMenuButton(BuildContext context){
+    final IconData arrowDown = const IconData(0xf3d0,
+          fontFamily: CupertinoIcons.iconFont,
+          fontPackage: CupertinoIcons.iconFontPackage);
+
+    return ConstrainedBox(
+      constraints: BoxConstraints(maxWidth: 30.0, maxHeight: 30.0),
+      child: PopupMenuButton(
+        padding: EdgeInsets.all(0.0),
+        offset: Offset(MediaQuery.of(context).size.width * 0.375, 30),
+        icon: Icon(
+          arrowDown,
+          color: Colors.white,
+          size: 20.0,
+        ),
+        onSelected: (key) => Scrollable.ensureVisible(
+            key.currentContext, 
+            duration: Duration(milliseconds: 500)
+        ),
+        color: kPrimaryColor,
+        itemBuilder: (BuildContext menuContext) => <PopupMenuEntry<GlobalKey>>[
```

Comment:
could this be done by looping/mapping through a list? Similar to this https://ptyagicodecamp.github.io/understanding-flutter-popupmenubutton-widget.html

Reply:
Not sure if I could map 2 lists to a widget at the same time. In this case, I'd have to map both global keys and section titles. Also, I don't think we should worry too much about this pop-up menu, since I'll be getting rid of it and making the sections collapsible instead in the next PR!